mm2src/coins/utxo/bch.rs

```diff
@@ -299,11 +317,13 @@ pub async fn bch_coin_from_conf_and_params(
     }
 
     let bchd_urls = params.bchd_urls;
+    let slp_tokens_infos = Arc::new(Mutex::new(HashMap::new()));
     let constructor = {
         move |utxo_arc| BchCoin {
             utxo_arc,
             slp_addr_prefix: slp_addr_prefix.clone(),
             bchd_urls: bchd_urls.clone(),
+            slp_tokens_infos: Arc::clone(&slp_tokens_infos),
```

Comment:
In the older Rust documentation `Arc::clone` was mentioned as a more idiomatic way: http://web.mit.edu/rust-lang_v1.26.0/arch/amd64_ubuntu1404/share/doc/rust/html/std/sync/struct.Arc.html#cloning-references.
> The Arc::clone(&from) syntax is the most idiomatic because it conveys more explicitly the meaning of the code. In the example above, this syntax makes it easier to see that this code is creating a new reference rather than copying the whole content of foo.
 
But it's not the case anymore: https://doc.rust-lang.org/std/sync/struct.Arc.html#deref-behavior 
> Arc<T>’s implementations of traits like Clone may also be called using fully qualified syntax. Some people prefer to use fully qualified syntax, while others prefer using method-call syntax.

I will change it to `slp_tokens_infos.clone()` as we use this approach everywhere else.